### PR TITLE
Differential Chassis fabrication

### PR DESCRIPTION
**Pull request by rg0699**
@Shubhanshu07

### CODE REVIEW

**Comment by agshubhs** [comment]
*@rg0699* 
### Following changes are needed
1. Make readme.md in place of doc file.
2. Use markdown syntax in readme file.
3. Also add images and videos in a separate folder.

**Comment by palnabarun** [comment]
@Shubhanshu07  Try to put inline reviews like I did just now.

File: README.md [hunk 1]
```diff
@@ -1 +1,50 @@
-# Differential-Chasis
\ No newline at end of file
+# MARS PROJECT
+# CHASSIS
```
Comment (palnabarun) [inline README.md]: Add blank new line between each block.

File: README.md [hunk 2]
```diff
@@ -1 +1,50 @@
...
+A differentially steered robot is similar to the differential gears used in automobiles in that both the wheels can have different rates of rotations, but unlike the differential gearing system, a differentially steered system will have both the wheels powered. Differential wheeled robots are used extensively in robotics, since their motion is easy to program and can be well controlled. Virtually all consumer robots on the market today use differential steering primarily for its low cost and simplicity.
+[differential](https://ai2-s2-public.s3.amazonaws.com/figures/2017-08-08/176dd7b3289267d8c693fb16f0d6fea480b5fad5/2-Figure2-1.png)
```
Comment (palnabarun) [inline README.md]: This would be a hyperlink. For rendering images, use `! [ ] ( )`

Comment (palnabarun) [inline README.md]: Do the same for each image in the Readme

File: README.md [hunk 3]
```diff
@@ -1 +1,50 @@
...
+# HARDWARE
+assembly of the chassis for the bot using:
+
+1)Caster wheel
```
Comment (palnabarun) [inline README.md]: Use Ordered list like this:
```
1. One
2. Two
```

File: README.md [hunk 4]
```diff
@@ -1 +1,50 @@
...
+
+7)3D printed parts
+# WORK
+The bot is made up of aluminium channels as the supporting framework on which 2 ATC Motors(12V), 2 Encoders are mounted at the back of the bot.Movement is based on two separately driven wheels placed on either side of the robot body.Each wheel is connected to the motor via shaft and coupling on which a spur gear is mounted. Another spur gear is mounted on the shaft connnected to encoder. Encoder are placed such that both the shafts are kept parallel so that the gears are mated.
+The motion of the bot is then programmed and controlled using arduino which is mounted on the chassis connected to the encoder. thus encoder gets the input accordingly and rotates the gear ar per the requirement which then rotates the gear connected to motor and the wheel. Therefore each wheel can have a different relative rate of rotation which is responsible for the change in direction of the bot.
+Another wheel called castor wheel is mounted at the front of the bot. This wheel is kept free and can move in any direction and act as an supporting wheel. 
+
+## WORK LEFT:
+
+-Mounting of 3D printed gears on both sides of bot.
```
Comment (palnabarun) [inline README.md]: Add a space between `-` and the content for an unordered list.

File: README.md [hunk 5]
```diff
@@ -1 +1,50 @@
...
+
+[arduino](https://cf1.s3.souqcdn.com/item/2013/10/11/61/86/78/0/item_XL_6186780_3246529.jpg)
+
+7)3D printed parts
+# WORK
+The bot is made up of aluminium channels as the supporting framework on which 2 ATC Motors(12V), 2 Encoders are mounted at the back of the bot.Movement is based on two separately driven wheels placed on either side of the robot body.Each wheel is connected to the motor via shaft and coupling on which a spur gear is mounted. Another spur gear is mounted on the shaft connnected to encoder. Encoder are placed such that both the shafts are kept parallel so that the gears are mated.
+The motion of the bot is then programmed and controlled using arduino which is mounted on the chassis connected to the encoder. thus encoder gets the input accordingly and rotates the gear ar per the requirement which then rotates the gear connected to motor and the wheel. Therefore each wheel can have a different relative rate of rotation which is responsible for the change in direction of the bot.
+Another wheel called castor wheel is mounted at the front of the bot. This wheel is kept free and can move in any direction and act as an supporting wheel. 
+
+## WORK LEFT:
```
Comment (palnabarun) [inline README.md]: Please do not put colons at the end of a header

**Review by palnabarun — COMMENTED** [review]
@rg0699  Please implement the reviews.

File: README.md [hunk 6]
```diff
@@ -1 +1,55 @@
...
+
+-Mounting of 3D printed gears on both sides of bot.
+
+-Mounting of Arduino and electronics work.
+
+# TEAM MEMBERS:
```
Comment (agshubhs) [inline README.md]: @rg0699  Write team members and mentors name separately and in alphabetical order with their Github link

**Review by agshubhs — CHANGES_REQUESTED** [review]
Add in actual images of the bot in the readme file as well as in a separate folder

**Review by agshubhs — APPROVED** [review]

File: README.md [hunk 7]
```diff
@@ -1 +1,71 @@
...
+A differentially steered robot is similar to the differential gears used in automobiles in that both the wheels can have different rates of rotations, but unlike the differential gearing system, a differentially steered system will have both the wheels powered. Differential wheeled robots are used extensively in robotics, since their motion is easy to program and can be well controlled. Virtually all consumer robots on the market today use differential steering primarily for its low cost and simplicity.
+![differential](https://upload.wikimedia.org/wikipedia/commons/d/d3/Tanklike.png)
+
+# HARDWARE
+assembly of the chassis for the bot using:
+
+1.Caster wheel
```
Comment (palnabarun) [inline README.md]: Please put a space between the number and the text.

**Review by palnabarun — COMMENTED** [review]
@rg0699 Please edit this.

**Comment by palnabarun** [comment]
The PR Looks Good to me. Please make the above suggested change and you are good to go. @rg0699

File: README.md [hunk 8]
```diff
@@ -1 +1,71 @@
-# Differential-Chasis
\ No newline at end of file
+# MARS PROJECT
+
+# CHASSIS
+A chassis is the internal framework of an artificial object, which supports the object in its construction and use. An example of a chassis is a vehicle frame, the underpart of a motor vehicle, on which the body is mounted.
+In an electronic device, the chassis consists of a frame or other internal supporting structure on which the circuit boards and other electronics are mounted.
+
+# DIFFERENTIAL CHASSIS
```
Comment (agshubhs) [inline README.md]: short the content of this paragraph and try to write these paragraph in points or bullets .

**Comment by palnabarun** [comment]
Looks Good To Me.

Squashing and Merging.

**Comment by palnabarun** [comment]
Good work. @rg0699 

@Shubhanshu07  Thank you for moderating the PR.

**Comment by rg0699** [comment]
thanks @palnabarun and @Shubhanshu07 sir for your guide.

**Comment by agshubhs** [comment]
nice work @rg0699

**Pull request merged**